### ISSUE TITLE
Request Timeout time does not increase than 15s

### ISSUE THREAD

**Issue by kahootali** [istio/istio#1888]
I am using Istio 0.2.12 and Kubernetes 1.8.1.
I have a service which returns quite a big response and when I load test it by calling a lot of calls in few seconds it takes more time than 15 seconds but the default timeout is 15s . So I have tried to make my own routerule 
```
apiVersion: config.istio.io/v1alpha2
kind: RouteRule
metadata:
  name: mysvc-routerule
spec:
  destination:
    name: mysvc
    namespace: istio-try
  httpReqTimeout:
    simpleTimeout:
      timeout: 25s
```
But it doesn't work, it still timeouts at 15s, where as if I decrease the timeout period to 5s, it works and timeouts at 5s rather than 15s. Any way I can configure it?

**Comment by GregHanson** [comment]
@kahootali can you reproduce the issue in 0.5.1 or later versions of istio?

**Comment by GregHanson** [comment]
as documented here:
https://www.envoyproxy.io/docs/envoy/v1.5.0/api-v1/route_config/route.html#config-http-conn-man-route-table-route-timeout

> timeout_ms
(optional, integer) Specifies the timeout for the route. If not specified, the default is 15s. Note that this timeout includes all retries. See also x-envoy-upstream-rq-timeout-ms, x-envoy-upstream-rq-per-try-timeout-ms, and the retry overview.

if this is still present in 0.5.1 or later, envoy may be enforcing an upper bound on the timeout

**Comment by frankbu** [comment]
This still seems to be an issue in 0.8. See #6860

**Comment by frankbu** [comment]
Fixed in Istio 1.0.0
